001-public-stream-base.md

```diff
@@ -0,0 +1,166 @@
+# Make C++ Stream APIs public
+
+Author: Fedor Indutny <fedor@indutny.com>
+Status: draft
+Date: Thu Dec 17 2015 19:39:46 GMT-0500 (EST)
```

Comment:
Consider doing this how is done in LEPS (https://raw.githubusercontent.com/libuv/leps/master/001-remove-unix-support.md). As this is now all these entries will be placed on the same line.